### PR TITLE
Actions: bumped python to 3.8.3 and fix for mergemodules

### PR DESCRIPTION
**Pull request by fdennis**

### CODE REVIEW

**Comment by dagheyman** [comment]
Builds doesn't seem to finish

**Comment by fdennis** [comment]
> Builds doesn't seem to finish

Yeah. Probably nothing serious. I have restarted them

File: .github/workflows/windows-x64.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: windows 64 bit
 on: [push, pull_request]
 
 env:
-  PYTHON_VER: '3.8.2'
+  PYTHON_VER: '3.x'
```
Comment (dagheyman) [inline .github/workflows/windows-x64.yml:6]: I guess this could brake when the action moves to 3.9 but the variable below is still 3.8 ?

Comment (fdennis) [reply .github/workflows/windows-x64.yml]: Yeah well I am just trying random stuff right now to see if something will work. But you are correct

**Pull request merged**